scripts/boot-master/load-image.sh

```diff
@@ -73,19 +73,18 @@ for image_location in ${locations[@]} ; do
 
     fi
 
-    # Download the file using auth if provided
-    echo "Downloading ${image_location}" >&2
-    echo "This can take a very long time" >&2
-    wget -nv --continue ${username:+--user} ${username} ${password:+--password} ${password} \
-     -O ${sourcedir}/${filename} "${image_location}"
+    if [[ ${filename} =~ .*$(uname -m).* ]]; then
```

Comment:
Since this executes on the boot node, would excluding download of images not for the same platform as boot node not break multi-arch support?